java/images/rhel/README.md

```diff
@@ -21,15 +21,45 @@ The following environment variables can be used to influence the behaviour of th
 
 The run script can be influenced by the following environment variables:
 
-* **JAVA_OPTIONS**  Options that will be passed to the JVM.  Use it to set options like the max JVM memory (-Xmx1G).
-* **JAVA_ENABLE_DEBUG**  If set to true, then enables JVM debugging
-* **JAVA_DEBUG_PORT** Port used for debugging (default: 5005)
-* **JAVA_AGENT** Set this to pass any JVM agent arguments for stuff like profilers
-* **JAVA_MAIN_ARGS** Arguments that will be passed to you application's main method.  **Default:** the arguments passed to the `bin/run` script.
-* **JAVA_MAIN_CLASS** The main class to use if not configured within the plugin
-
-The environment variables are best set in `.sti/environment` top in you project. This file is picked up bei S2I
-during building and running.
+* **JAVA_APP_DIR** the directory where the application resides. All paths in your application are relative to this directory.
+* **JAVA_LIB_DIR** directory holding the Java jar files as well an optional `classpath` file which holds the classpath. Either as a single line classpath (colon separated) or with jar files listed line-by-line. If not set **JAVA_LIB_DIR** is the same as **JAVA_APP_DIR**.
+* **JAVA_OPTIONS** options to add when calling `java`
+* **JAVA_MAX_MEM_RATIO** is used when no `-Xmx` option is given in `JAVA_OPTIONS`. This is used to calculate a default maximal Heap Memory based on a containers restriction. If this script is used in a Docker container without any memory constraints for the container then this option has no effect. If there is a memory constraint then `-Xmx` is set to a ratio of the container available memory as set here. The default is 85 which means 85% of the available memory is used as an upper boundary. You can skip this mechanism by setting this value to 0 in which case no `-Xmx` option is added on behalf od this script.
```

Comment:
In fact, https://github.com/rhuss/run-java-sh/blob/master/fish-pepper/run-java-sh/readme.md contains already the corrected size, so maybe one just need to re-run fish-pepper ?